scripts/run_frontend.sh

```diff
@@ -0,0 +1,2 @@
+cd frontend
```

Comment:
This will only work if this bash script is called from the root gradio folder. Might want to put a comment on top clarifying that.